test/test_quantized.py

```diff
@@ -1809,6 +1838,32 @@ def test_qnnpack_relu(self, X):
             qY = torch.quantize_per_tensor(Y, scale=scale, zero_point=zero_point, dtype=torch_type)
             self.assertEqual(qY, qY_hat)
 
+    """Tests the correctness of the quantized::qnnpack_tanh op."""
+    @given(X=hu.tensor(shapes=hu.array_shapes(1, 5, 1, 5),
+                       qparams=hu.qparams(dtypes=torch.quint8)))
+    def test_qnnpack_tanh(self, X):
+        # Note: In QNNPACK the output scale and zero_point can only be
+        #       2.0/256, 128 respectively, as it uses a LUT with 256 bins.
+        X, (scale, zero_point, torch_type) = X
+        X = torch.from_numpy(X)
+        qX = torch.quantize_per_tensor(X, scale=scale,
+                                       zero_point=zero_point,
+                                       dtype=torch_type)
+
+        # Floating point reference
+        Y = torch.tanh(X)
+        qY = torch.quantize_per_tensor(Y, scale=1.0 / 128, zero_point=128,
+                                       dtype=torch.quint8)
+        with override_quantized_engine('fbgemm'):
```

Comment:
You don't need an override here. It should call the server side implementation of quantized tanh by default. 
Can you also call it something other than fbgemm? (`qYserver` maybe)